File: Algoritmo(C)/Alg_S8_5.c
#include <stdio.h>
int main(){
    //variables
    int vetor[10], i, cont = 0;
    //input
    for(i = 0; i <= 9; i++){
        printf("Digite um valor para o vetor %d/10: ",(i + 1));
        scanf("%d",&vetor[i]);
    }
    //process + output
    for(i = 0; i <= 9; i++){
        if(vetor[i] >= 50){
            printf("O numero %d esta na posicao %d\n",vetor[i],(i + 1));
            cont = 1;
        }
    }
    if(cont < 1){
        printf("Nenhum numero maior que 50 foi digitado");
    }
}

File: Algoritmo(Python)/Alg_S8_5.py
#variables
vetor = []
maior = False
#input + process
for(i) in range(0, 10):
    num = int(input("Digite um número para o vetor {0}/9: ".format(i)))
    vetor.append(num)
for(i) in (vetor):
    if(i > 50):
        print("Valor {0} maior que 50 na posição {1}".format(i, vetor.index(i)))
        maior = True
if(maior == False):
    print("Nenhum número maior que 50 foi digitado")